### DISCUSSION TITLE
How to display boundary attributes?

### DISCUSSION THREAD

**Discussion by QiangWei-Wade** [GLVis/glvis#117] [Q&A] [answered]
Hi everyone,

Is there a key I can press to display or color boundary attributes when using glvis -m xxx.mesh to display a 3D mesh?

Thanks,
Wade

**Comment by v-dobrev** [comment, marked as answer]
You can press F9/F10 to cycle forward/backward through the boundary attributes. F8 will let you toggle a list of boundary attributes you type in the terminal.

To view the boundary attributes by color you can use the `mesh-explorer` miniapp from `miniapps/meshing`, choosing option `'b'` from its menu.

**Comment by QiangWei-Wade** [comment]
@v-dobrev 

Got it. Thank you!
